dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -841,6 +845,22 @@ private String computeWorkflowName(Workflow workflow) {
         return workflow.getWorkflowName();
     }
 
+    private String computeWorkflowPhrase(Class<?> workflowType, Workflowish workflow) {
```

Comment:
What's a phrase in this context? Looks like the entry type and path?
Could use code comment for the three methods or more descriptive method name.